### ISSUE TITLE
Found your wallet

### ISSUE THREAD

**Issue by kev23777** [xg590/tutorials#2]
Hello, 

In not 100% that this is the correct Guo Xiaokang from New York. We're you born 05/28/1990?

I found your little sleeve wallet outside the Midway Museum.

Trying to figure out how to get it back to you.

**Comment by kev23777** [comment]
I called your credit card company. They told me to take your wallet to the police, but apparently all the police stations close at 6pm so you wouldn't have been able to get your wallet until tomorrow at the earliest.

You had a Wyndham room key. So we went to the Wyndham San Diego Bayside. The concierge said that you weren't staying at that Wyndham, but they would call all of the other Wyndhams and find you.

So we left your wallet with the concierge at
1335 N. Harbor Dr. San Diego, CA 92101

Their phone number is 619-232-3861.

Hopefully they find you at whichever Wyndham that your are staying at.
